### PR TITLE
Add Clear Logs Button

### PR DESCRIPTION
**Pull request by hahmadia**
In regards to https://github.com/microsoft/TypeScript-Website/issues/450 , I've added a clear log button to make it more clear that functionality is present. We would still be missing the `Filter` option as mentioned in the issue.

**Before**
image

**After**
image

### CODE REVIEW

**Comment by hahmadia** [comment]
Hi @orta ,

If you have any sort of style guide/file/recommendation in mind, please let me know. I’m open to doing the styling changes myself if you are busy with other work.

Thanks!

**Comment by orta** [comment]
Sure, I think this is probably what I was going to aim for the cancel button: 

Screen Shot 2021-03-08 at 7 57 30 PM

The SVG for the 

```svg
<svg width="13" height="13" viewBox="0 0 13 13" fill="none" xmlns="http://www.w3.org/2000/svg">
<circle cx="6" cy="7" r="5" stroke="black" stroke-width="2"/>
<line x1="0.707107" y1="1.29289" x2="11.7071" y2="12.2929" stroke="black" stroke-width="2"/>
</svg>
```

Today there is a margin on the bottom of the sidebar, I'd use a negative margin to work around it

**Comment by hahmadia** [comment]
@orta 

Let me know what you think. Tried my best to duplicate your design. Since it included filtering, I decided I would add it in this PR.

**Initially**
image


**Filter something**
image


**Delete filtered text (scrolls back down to bottom of logs)**
image

When you click clear, it sets the filter textbox to blank and clears your logs.

File: packages/playground/src/sidebar/runtime.ts [hunk 1]
```diff
@@ -101,22 +165,13 @@ function rewireLoggingToElement(
       const eleContainerLog = eleOverflowLocator()
       allLogs.push(`${prefix}${output}<br>`);
       eleLog.innerHTML = allLogs.join("<hr />")
-      const scrollElement = eleContainerLog.parentElement
-      if (autoScroll && scrollElement) {
-        scrollToBottom(scrollElement)
+      if (autoScroll && eleContainerLog) {
+        eleContainerLog.scrollTop = eleContainerLog.scrollHeight
```
Comment (orta) [inline packages/playground/src/sidebar/runtime.ts:169]: Hah, well that's a drastic simplification - looks like mechanically it seems to be the same 👍🏻

**Review by orta — APPROVED** [review]

**Comment by orta** [comment]
Yeah, nice one! 

This looks good to me 👍🏻 - will test it out on staging today and deploy if it feels good

**Pull request merged**